policy/release/lib/attestations.rego

```diff
@@ -65,7 +67,15 @@ unmarshal(raw) = value {
 
 # (Don't call it test_results since test_ means a unit test)
 results_from_tests = results {
```

Comment:
Do we need to support when a mix of `HACBS_TEST_OUTPUT` and `TEST_OUTPUT` are used?

Reply:
I was thinking let's not do that. I think it will make this patch a bit more complicated, and it seems unlikely it would happen.

I guess if it does happen, the impact is that ec will report a "required test not found" violation and someone will need to update their task definitions.

Wdyt?

Reply:
Users can update the tasks in their pipelines individually, so it's not that much of a corner case. But maybe it's not something we need.